src/docfx/publish/PublishModelBuilder.cs

```diff
@@ -79,6 +82,7 @@ orderby monikerGroup
                 _config.Name,
                 _config.Product,
                 _config.BasePath.ValueWithLeadingSlash,
+                _templateEngine.GetThemeBranch(),
```

Comment:
Is it suffice enough to say:
```suggestion
                _config.Template.Branch,
```